### DISCUSSION TITLE
can i see to TPS?

### DISCUSSION THREAD

**Discussion by anstnsp** [SamYuan1990/distributedtape#7] [Q&A] [answered]
like the title above, i want to see TPS but i did not find anywhere T.T

**Comment by SamYuan1990** [comment, marked as answer]
by default, this project starts Tape in daemon set mode,  and https://github.com/Hyperledger-TWGC/tape/issues/94 is still open there. 
so two options.
1. running tape in number module, ref https://github.com/SamYuan1990/distributedtape/blob/main/roles/observer/tasks/main.yml#L7 and https://github.com/Hyperledger-TWGC/tape/blob/alpha/test/integration-test.sh#L117-L119

2. or try with a pr on tape for https://github.com/Hyperledger-TWGC/tape/issues/94 and pr target to alpha branch.
